samcli/lib/iac/cfn/cfn_iac.py

```diff
@@ -1,33 +1,182 @@
 """
 Provide a CFN implementation of IaCPluginInterface
 """
-from typing import List
 
-from samcli.lib.iac.plugins_interfaces import IaCPluginInterface, SamCliProject, Stack, LookupPath
+# pylint: skip-file
+# TODO: stop skipping once we remove the implementation in samcli/lib/iac/cfn_iac.py
```

Comment:
I think this one is not in `develop` and we don't need pylint skip here if it's only due to the duplication.

Reply:
Good call out.